src/linker/ref/Linker/AssemblyResolver.cs

```diff
@@ -0,0 +1,51 @@
+﻿//
```

Comment:
Let's go with minimal set so APIs like that should not be exposed

Reply:
Removed this.